Voycar.Api.Web.Tests.Integration/Users/Post_Register/Tests.cs

```diff
@@ -0,0 +1,327 @@
+namespace Voycar.Api.Web.Tests.Integration.Users.Post_Register;
+
+using Context;
+using Entities;
+using Microsoft.EntityFrameworkCore;
+using R = Features.Users.Endpoints.Post.Register;
+
+
+public sealed class State : StateFixture
+{
+    public Guid Id { get; set; }
+
+    public static Guid RoleId { get; set; }
+    public const string RoleName = "member";
+
+    public static Guid PlanId { get; set; }
```

Comment:
Die Attribute nicht static, sollten ganz normale Klassenattribute sein
```suggestion
    public Guid RoleId { get; set; }
    public const string RoleName = "member";

    public Guid PlanId { get; set; }
```